react-debounce-input/build.boot

```diff
@@ -0,0 +1,30 @@
+(def +lib-version+ "3.0.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies   [['cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
In addition to deps-cljs react require, this package should also have a dependency on `cljsjs/react`.